serverless/integrations/gpu-resources.adoc

```diff
@@ -23,20 +23,19 @@ You can specify a GPU resource requirement when you create a Knative service usi
 . Create a service.
 . Set the GPU resource requirement limit to `1` by using `nvidia.com/gpu=1`:
 +
-
 [source,terminal]
 ----
 $ kn service create hello --image docker.io/knativesamples/hellocuda-go --limit nvidia.com/gpu=1
 ----
-
 +
 A GPU resource requirement limit of `1` means that the service has 1 GPU resource dedicated.
 Services do not share GPU resources. Any other services that require GPU resources must wait until the GPU resource is no longer in use.
 +
 A limit of 1 GPU also means that applications exceeding usage of 1 GPU resource are restricted.
 If a service requests more than 1 GPU resource, it is deployed on a node where the GPU resource requirements can be met.
 
-.Updating GPU requirements for a Knative service using `kn`
+[id="gpu-resources-updating-requirements"]
```

Comment:
This whole assembly is a bit weird. There is a procedure above it that should be in it's own module, and then it seems like this section should be its own module too (with the .Procedure heading, etc. - and the command indented properly under the step). 

A bit out of scope for what you're doing here, other than the fact that you moved some other content into its own module. But it's fine if you want to take a look at it for a future update!

Reply:
Made some modifications to improve this, lmk what you think 🙂 thanks for the suggestion!